tests/test_sqlalchemy_create_table.py

```diff
@@ -0,0 +1,64 @@
+# -*- coding: utf-8; -*-
```

Comment:
Put this in a separate cause more will come later when we add more dialect kwargs to support more features. Also because I don't like to mix tests that need external resources with some that don't. Like the need for an actual connection to Athena, that here we don't need. But that's just my habits.

Reply:
Please don't separate the tests, just add the following simple test to test_sqlalchemy_athena.py and it should be fine.
```
    def test_create_table(self):
        table = Table(
            "test_create_table",
            MetaData(),
            Column("column_name", String),
            schema="test_schema",
            awsathena_location="s3://path/to/test_schema/test_create_table",
            awsathena_compression="SNAPPY",
        )
        dialect = AthenaDialect()
        actual = CreateTable(table).compile(dialect=dialect)
        self.assertEqual(
            str(actual),
            textwrap.dedent(
                """
                CREATE EXTERNAL TABLE test_schema.test_create_table (
                    column_name VARCHAR
                )
                STORED AS PARQUET
                LOCATION 's3://path/to/test_schema/test_create_table'
                TBLPROPERTIES ('parquet.compress'='SNAPPY')

                """
            ),
        )
```
Please try to match the way you write your existing tests. There is no need to use pytest.fixture.